go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -137,7 +139,8 @@ var tableACLRoles = map[PlanType]tableacl.Role{
 	PlanInsertSubquery: tableacl.WRITER,
 	PlanDDL:            tableacl.ADMIN,
 	PlanSelectStream:   tableacl.READER,
-	PlanOther:          tableacl.ADMIN,
+	PlanOther:          tableacl.READER,
```

Comment:
I actually think we should rename PlanOther and Other ast type to make them more obvious that it will be allowed for anyone with READER access.